tests/integration/test_dbapi_integration.py

```diff
@@ -231,7 +231,7 @@ def test_experimental_python_types_with_connection_and_cursor(
 
 
 def test_decimal_query_param(trino_connection):
-    cur = trino_connection.cursor(experimental_python_types=True)
+    cur = trino_connection.cursor(legacy_primitive_types=False)
```

Comment:
Please omit the parameter as the default is tested in `test_legacy_primitive_types_with_connection_and_cursor`

Reply:
omitted